src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -160,6 +163,9 @@ private void verify(Configuration config, String filePath, String[] expected,
             final IndentComment... linesWithWarn) throws Exception {
         final Checker checker = createChecker(config);
         checker.addListener(new IndentAudit(linesWithWarn));
+        checker.setLocaleCountry(Locale.ROOT.getCountry());
+        checker.setLocaleLanguage(Locale.ROOT.getLanguage());
+        checker.finishLocalSetup();
```

Comment:
@romani To do a 3-way validation for Indentation (expected vs message vs input file), the only way we could get the required indentation levels was to parse the numbers from the message directly. There is no hook in the check that lets us see these values before the message is created.
Here is the issue: https://github.com/checkstyle/checkstyle/issues/3105

The only way we can fix this and get the numbers outside the message is probably to either override `IndentationCheck` in test or to try and use powermock to call a test method before the check's method. We would have to target `IndentationCheck.indentationLog`.